src/test/user.spec.js

```diff
@@ -17,7 +22,7 @@ describe('Test user login, signup and account verification', () => {
   let validToken;
 
   describe('POST /auth/signup', () => {
-    it('it should create a user account', done => {
+    it('should create a user account', done => {
```

Comment:
Nice test set up 👍for a test-driven development (TDD) approach, however, there is less confidence in this test as there are no assertions to test for the properties (behaviour) of a user (username, email, role, status) as shown in the response body.

This applies to the rest of the tests. 
Going forward, there is room for improvement on these tests in order to give confidence in the feature implemented.